FilmFestivalLoader/Shared/web_tools.py

```diff
@@ -7,8 +7,14 @@
 """
 
 from html.parser import HTMLParser
+import urllib.request
+import urllib.error
+from urllib.parse import quote                                                                                                                                                                
 
 
+def uripath_to_iripath(path):
+    return quote(path)
+
```

Comment:
Good research!
Please consider to import more consistently from `url lib`, e.g. by using `import url lib.parse`.

Reply:
Done.
Plus changed the other "from .. import" statements into "import".